Client/app/src/main/java/ru/hse/client/groups/GroupRequests.kt

```diff
@@ -187,3 +188,99 @@ fun createGroupRequest(groupName: String, groupPassword: String, activity: Activ
 
     countDownLatch.await()
 }
+
+fun acceptInvite(groupId: Long, activity: Activity, okHttpClient: OkHttpClient) : Boolean {
+    val URlCreateGroup: String =
+            ("http://" + ContextCompat.getString(activity, R.string.IP) + "/invitations/accept/user").toHttpUrlOrNull()
+                    ?.newBuilder()
+                    ?.addQueryParameter("groupId", groupId.toString())
+                    ?.addQueryParameter("userId", user.getId().toString())
+                    ?.build()
+                    .toString()
+
+    val requestBody: RequestBody = RequestBody.create(null, ByteArray(0))
+
+    val request: Request = Request.Builder()
+            .url(URlCreateGroup)
+            .post(requestBody)
+            .header("Authorization", "Bearer " + user.getUserToken())
+            .build()
+
+    Log.i("GroupRequest", "Request has been sent $URlCreateGroup")
+
+    val somethingWentWrongMessage: String = "Something went wrong, try again"
+    val countDownLatch = CountDownLatch(1)
+    var isSuccess = false
+    okHttpClient.newCall(request).enqueue(object : Callback {
+        override fun onFailure(call: Call, e: IOException) {
+            Log.e("GroupRequest", e.toString() + " " + e.message)
+            Handler(Looper.getMainLooper()).post {
+                Toast.makeText(activity, somethingWentWrongMessage, Toast.LENGTH_SHORT).show()
+            }
+            countDownLatch.countDown()
+        }
+
+        override fun onResponse(call: Call, response: Response) {
+            Log.i("Info", response.toString())
+            if (response.isSuccessful) {
+                isSuccess = true
+                Log.i("GroupRequest", "user accept invite")
```

Comment:
May be we will write user id or something to know with which user we work in request? Soon logs like this may cause problems